staging/src/k8s.io/kubectl/pkg/cmd/rollout/rollout_status.go

```diff
@@ -168,6 +168,13 @@ func (o *RolloutStatusOptions) Run() error {
 		Flatten().
 		Do()
 
+	info, _ := r.Infos()
```

Comment:
I think you can use either `Infos` or `Visit` but not both of them, since they have effect on the other one in terms of the results. And in that case, we are using `Visit` function and we might continue using it.

What about if there is any resource, `Visit` function will be called and we can use this via flag(`resourceFound = true` in `Visit` function)? or any idea hasn't come to my mind?